borgmatic/config/schema.yaml

```diff
@@ -1265,3 +1265,28 @@ properties:
                     Umask used when executing hooks. Defaults to the umask that
                     borgmatic is run with.
                 example: 0077
+            snapper:
```

Comment:
Snapper is the *only* tool that does something like this (AFAIK), but not *the* tool (IMO). ZFS support is [apparently](https://github.com/openSUSE/snapper/issues/145) somewhat on the way but seems pretty stalled.
So it seems possible for something like `zfs_snapshots` to make sense in the future.

I disagree with your conclusion though: I would call it `snapper` and `zfs_snapshots` since I don't find these suffixes particularly helpful. I would also point to `cronitor`, `healthchecks`, `pagerduty`, ... not having a common suffix (e.g. `_monitoring`).

But ultimately this is your thing, and I am fine with whatever you want this to be named.

Reply:
Nah, I don't feel strongly here. IIRC for the database hooks, the `_databases` suffix was less to serve as a crappy namespace and more because of what each one is listing is.. databases! But in the case of your hook, you're not listing snapshots, per se. So :shrug:.

Reply:
> Snapper is the only tool that does something like this

That's probably correct when it comes to root snapshots.
For general snapshots, [btrbk](https://github.com/digint/btrbk) comes to mind